webpack.config.js

```diff
@@ -1,174 +1,68 @@
-const fs = require('fs');
 const path = require('path');
-const webpack = require('webpack');
-const HtmlWebpackPlugin = require('html-webpack-plugin');
-const HtmlBeautifyPlugin = require('@nurminen/html-beautify-webpack-plugin');
```

Comment:
In production mode use the `minfy` of HTML (good practice), not `beautify` to avoid some issues with spaces in HTML.